libs/data/bbhnet/data/distributions.py

```diff
@@ -71,3 +71,26 @@ def __call__(self, N: int) -> torch.Tensor:
         if self.low is not None:
             x = torch.clip(x, self.low)
         return x
+
+
+class VolumeDistributedSNR:
```

Comment:
For the sake of consistency, it might make sense to have the name of this class describe the distribution itself, e.g. `PowerLaw` or something. That SNR distributions tend to follow it is a useful application, but not necessarily a defining attribute of the distribution.